doc/source/whatsnew/v0.24.0.txt

```diff
@@ -26,6 +26,33 @@ Backwards incompatible API changes
 
 .. _whatsnew_0240.api.datetimelike:
 
+Tick DateOffset Normalize Restrictions
+--------------------------------------
+
+Creating a ``Tick`` object (:class:``Day``, :class:``Hour``, :class:``Minute``,
+:class:``Second``, :class:``Milli``, :class:``Micro``, :class:``Nano``) with
+`normalize=True` is no longer supported.  This prevents unexpected behavior
+where addition could fail to be monotone or associative.
+
+.. ipython:: python
+
+    ts = pd.Timestamp('2018-06-11 18:01:14')
```

Comment:
show ts and tic here

Reply:
Will do.  Does showing the tic here work given that the line that defines it will raise in the new version?